lib/task/propel/propelBuildNestedSetTask.class.php

```diff
@@ -114,6 +111,7 @@ protected function configure()
             new sfCommandOption('env', null, sfCommandOption::PARAMETER_REQUIRED, 'The environment', 'cli'),
             new sfCommandOption('connection', null, sfCommandOption::PARAMETER_REQUIRED, 'The connection name', 'propel'),
             new sfCommandOption('exclude-tables', null, sfCommandOption::PARAMETER_OPTIONAL, 'Exclude tables (comma-separated). Options: information_object, term, menu'),
+            new sfCommandOption('no-index', null, sfCommandOption::PARAMETER_OPTIONAL, 'Do not re-index after building nested set', false),
```

Comment:
I'd do it the other way around to maintain the current behavior. Like in:

https://github.com/artefactual/atom/blob/7d52ca72de98df50b64f86bcd9f2b7a0fc805113/lib/task/digitalobject/digitalObjectLoadTask.class.php#L252

Then you can set it to true in:

https://github.com/artefactual/atom/blob/7d52ca72de98df50b64f86bcd9f2b7a0fc805113/lib/task/import/csvImportTask.class.php#L1010